resources/js/directives/projects.js

```diff
@@ -378,13 +374,8 @@ angular.module('coala')
                         $scope.projects_url_dict[value["url"]] = key
                     });
 
-                    $scope.sortProjects();
-
```

Comment:
Why are we doing this though?

Reply:
From what I understood, sortProjects function sorts the projectList by latest. Here, in the projectRequest function, sortProjects function reorders the list before making query of URL requested. So, we end up on some different project. This sortProjects call is required to the maintain the list after requesting URL to navigate to left and right. I missed that point, so we can make this call after the query is done.